sites/partners/__tests__/pages/users/terms.test.tsx

```diff
@@ -51,9 +51,9 @@ describe("User Terms", () => {
         /I have reviewed the(.*) for this Website, as that term is defined in the Terms of Use, and agree to comply with all requirements described therein that relate to my use as a Professional Partner or Local Government. If I am agreeing to comply with the Terms of Use on behalf of a Professional Partner or Local Government, I warrant that I am authorized to enter into agreements such as the Terms of Use on behalf of such Professional Partner or Local Government./
       )
     ).toBeInTheDocument()
-    const termsOfUseLink = getByText("Terms of Use", { selector: "a" })
-    expect(termsOfUseLink).toBeInTheDocument()
-    expect(termsOfUseLink).toHaveAttribute(
+    const termsOfUseLink = getAllByText("Terms of Use", { selector: "a" })
+    expect(termsOfUseLink[0]).toBeInTheDocument()
```

Comment:
nit: rather than testing with `toBeInTheDocument` you should check the length of the termsOfUseLinks to be 2